### DISCUSSION TITLE
Tauri Webdriver support for macOS

### DISCUSSION THREAD

**Discussion by aleenadavy90** [tauri-apps/tauri#7045] [Q&A]
Hi,

We have been setting up e2etesting for our Tauri based app using Selenium in both Windows and Linux platform. Is there a timeline on when [tauri-driver](https://crates.io/crates/tauri-driver) support will be available for macOS? Thanks!

**Comment by FabianLars** [comment]
There is no timeline/ETA, especially since we're lacking macOS developers.
